lib/hpcbase.pm

```diff
@@ -16,6 +16,14 @@ use version_utils 'is_sle';
 use lockapi;
 use Utils::Logging 'save_and_upload_log';
 
+our $slurm_pkg;
+
+BEGIN {
+    if (get_var('HPC', '') =~ /slurm/) {
+        $slurm_pkg = get_var('SLURM_VERSION', '') ? 'slurm_' . get_var('SLURM_VERSION') : 'slurm';
+    }
+}
+
```

Comment:
Please do standard function definition (like we do everywhere else) which will have one argument for version and will return package name and not this approach.